tests/v1/generation/test_batch_invariance.py

```diff
@@ -111,7 +114,7 @@ def test_v1_generation_is_deterministic_across_batch_sizes_with_needle():
         # Engine with bs=1 behavior
         llm_bs1 = LLM_with_max_seqs(
             model=model,
-            max_num_seqs=1,
+            max_num_seqs=128,
```

Comment:
Why do we change to 128 here?

Reply:
this is necessary due to a bug in the initialization somewhere (requiring at least 10 IIUC - something with the cudagraph warmup).  it's orthogonal to what is being tested

Reply:
Make sense, thanks!